go/engine/pgp_import_key.go

```diff
@@ -234,6 +240,17 @@ func (e *PGPKeyImportEngine) Run(m libkb.MetaContext) (err error) {
 	return nil
 }
 
+func (e *PGPKeyImportEngine) checkRandomPassword(mctx libkb.MetaContext) error {
+	random, err := libkb.LoadHasRandomPw(mctx, keybase1.LoadHasRandomPwArg{})
+	if err != nil {
+		return err
+	}
+	if random {
+		return libkb.NewPushSecretWithoutPasswordError("A password is required to upload secret keys to the server.")
```

Comment:
After second thought, I don't think we need specific error type here, considering it will not be exportable anyway.

Also let's change the message to "You need to set your password first before uploading secret keys"